### ISSUE TITLE
Great work, Thanks Tom!

### ISSUE THREAD

**Issue by wootwoot1234** [ReactExtensionManager/ReactExtensionManager#3]
Great work, Thanks Tom!

**Comment by ide** [comment]
I'm excited to see this coming together. I think it will make a big difference in the React Native ecosystem. Once we have a couple native modules published and working, @TomMcHugh you should share this in the React Native group (https://www.facebook.com/groups/react.native.community/).

**Comment by tommymchugh** [comment]
Definitely. Happy to see it benefit the react native community (and it sure is helping me with my projects right now!)
